glide.yaml

```diff
@@ -6,7 +6,8 @@ import:
 - package: k8s.io/apiserver
   version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.13.4
+  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
+  version: 6e2b3eb1b401beaf836f640f73f730f6da7aff2f # pinned to get nullable patch
```

Comment:
I would track a branch instead.  Also, why do this when the ugly should be hidden in the new repo?

Reply:
Tracking branch now with TODO about moving this to new repo once @damemi is ready.